test/libsolidity/semanticTests/variables/transient_state_address_variable_members.sol

```diff
@@ -0,0 +1,15 @@
+contract D {}
+
+contract C {
+    address transient a;
+    function f() public returns (bytes32) {
+        D d = new D();
+        a = address(d);
+        return a.codehash;
```

Comment:
`.codehash` won't work on EOF. `.balance` would be more portable.

Reply:
I expected you'd actually send some ether to the contract so that the expected balance is non-zero :) Zero is the most likely thing we'd get if the test does not work so it makes it a bit less effective.

Reply:
tbh I thought about that but then could not figure a way of creating a contract with `balance` > 0.
I had a couple of tries at it but didn't came up with anything. How can we do that?

Reply:
https://github.com/ethereum/solidity/blob/develop/test/libsolidity/semanticTests/isoltestTesting/account.sol

Reply:
> https://github.com/ethereum/solidity/blob/develop/test/libsolidity/semanticTests/isoltestTesting/account.sol

@nikola-matic , I looked at that but it didn't help me setting up a balance for the contract...how would that work?

Reply:
Something like this should be fine:

```
contract C {
    address transient a;
    function f() public returns (uint) {
        a = msg.sender;
        return a.balance;
    }
    function g() public returns (uint) {
        return a.balance;
    }
}
// ====
// EVMVersion: >=cancun
// compileViaYul: false
// ----
// constructor() ->
// gas legacy: 59027
// gas legacy code: 70400
// account: 0 -> 0x1212121212121212121212121212120000000012
// balance: 0x1212121212121212121212121212120000000012 -> 1267650600228229401496703205376
// f() -> 1267650600228229401496703205376
// g() -> 0
```

Reply:
Ok, thanks. But I thought we could set an arbitrary value.

Reply:
I tried, but not sure if it's possible. E.g., the account address is hardcoded in `ExecutionFramework.cpp`...